__tests__/app.test.js

```diff
@@ -367,4 +367,70 @@ describe("POST /api/articles/:article_id/comments", () => {
         expect(body.message).toBe("Article not found");
       });
   });
+  test("should return a 400 code when passed an invalid article_id", () => {
+    return request(app)
+      .patch("/api/articles/six")
+      .send({ inc_votes: -50 })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.message).toBe("Invalid Data Type");
+      });
+  });
+  test("should return a 400 code when passed an invalid inc_votes value", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: "five" })
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.message).toBe("Invalid Data Type");
+      });
+  });
+});
+
+describe("PATCH /api/articles/:article_id", () => {
+  test('should return a 201 code and an article object with the "votes" property incremented by the "inc_votes" passed', () => {
```

Comment:
This is more likely to be a `200` code as we're not creating content but rather updating existing content